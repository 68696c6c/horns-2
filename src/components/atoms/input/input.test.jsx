/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Input from '.'

describe('Input', () => {
  it('should render as default', () => {
    const { container } = render(<Input id="example" name="example" />)
    expect(container).toMatchSnapshot()
  })
})
