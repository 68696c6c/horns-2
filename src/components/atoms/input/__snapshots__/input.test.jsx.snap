// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render as default 1`] = `
<div>
  <input
    class="control css-b5shpp"
    cursor="text"
    id="example"
    name="example"
    placeholder=""
    type="text"
  />
</div>
`;
